VAMobile/src/components/EncourageUpdate.tsx

```diff
@@ -88,7 +88,7 @@ export const EncourageUpdateAlert = () => {
   if (displayEU) {
     return (
       <Box mb={theme.dimensions.buttonPadding}>
-        <AlertBox title={t('encourageUpdate.title')} text={t('encourageUpdate.body')} border="warning">
+        <AlertWithHaptics variant="warning" header={t('encourageUpdate.title')} description={t('encourageUpdate.body')}>
           <Box>
             <Box my={theme.dimensions.gutter} accessibilityRole="button" mr={theme.dimensions.buttonPadding}>
               <Button onPress={onUpdatePressed} label={t('updateNow')} />
```

Comment:
Could be simplified moving these to the `primaryButton` and `secondaryButton` props to AlertWithHaptics.